src/style.css

```diff
@@ -1,3 +1,86 @@
+:root {
+  --color-primary: #00acdc;
+  /* nav__link settings without margin */
+  --nav__link__padding: 22.5px 0;
+  --nav__link__font-family: roboto, sans-serif;
+  --nav__link__font-size: 12px;
+  --nav__link__font-weight: 500;
+  --nav__link__text-decoration: none;
+  --nav__link__color: black;
+  --nav__link__text-transform: uppercase;
+  --nav__link__display: block;
```

Comment:
don't need to create variable for all styles, variables should be created only for duplicated styles

Reply:
They are repeated, if you need to change navigation styles you will have to change them 2 times